src/components/ListSelectionPage/ListSelectionPage.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { InjectAppServices } from '../../services/pure-di';
+import { Loading } from '../Loading/Loading';
+import { SubscriberListState } from '../../services/shopify-client';
+import { Promotional } from '../ContactPolicy/Promotional';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { useFetch } from '../../hooks/useFetch';
+
+const listsPerPage = 20;
+
+const ListRow = ({ list }) => {
+  return (
+    <tr>
+      <td aria-label="List Check">
+        <label className="dp-list-check">
+          <input type="checkbox" />
+          <span className="checkmark" />
+        </label>
+      </td>
+      <td aria-label="Name">
+        <span>{list.name}</span>
+      </td>
+      <td aria-label="Subscribers Amount">
+        <span>{list.amountSubscribers}</span>
+      </td>
+    </tr>
+  );
+};
+
+const ListTable = ({ lists }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const rows = [];
+  lists.forEach((list) => {
+    rows.push(<ListRow list={list} key={list.id} />);
+  });
+  return (
+    <table
+      className="dp-c-table"
+      aria-label="Selección de listas a excluir de la regla"
+      summary="Selección de listas a excluir de la regla"
+    >
+      <thead>
+        <tr>
+          <th>
+            <img
+              src={_('common.ui_library_image', { imageUrl: 'check-grey.svg' })}
+              alt="check list"
+            />
+          </th>
+          <th>Nombre de la lista</th>
+          <th>suscriptores</th>
+        </tr>
+      </thead>
+      <tbody>{rows}</tbody>
```

Comment:
It seems better to do it directly here: 
```
lists.map((list) => (
    <ListRow list={list} key={list.id} />);
)
```

I don't think there is any performance improvement, but it helps us separate presentation from events and how the data is prepared for display. Also, you avoid creating a variable